.vscode/settings.json

```diff
@@ -16,5 +16,9 @@
     },
     "python.analysis.extraPaths": [
         ".pixi/env/Library/python"
+    ],
+    "mypy-type-checker.importStrategy": "fromEnvironment",
+    "mypy-type-checker.args": [
+        "--config-file=${workspaceFolder}/mypy.ini"
```

Comment:
Did you double check that this is really necessary?

Reply:
You are right, no longer needed